### PR TITLE
Patient lab results should all print, with a patient banner and some margin

### PR DESCRIPTION
**Pull request by brandones**
Resolves Debbie's feedback on https://pihemr.atlassian.net/browse/UHM-6224 , except for the note about text wrapping.

[Lab 2012-04-13 2022-04-13.pdf](https://github.com/openmrs/openmrs-owa-labworkflow/files/8484864/Lab.2012-04-13.2022-04-13.pdf)

### CODE REVIEW

File: app/js/components/ConceptDisplay.jsx [hunk 1]
```diff
@@ -12,27 +12,19 @@ const {
 } = formValidations;
 
 class ConceptDisplay extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      concept: {},
-    };
-  }
```
Comment (brandones) [inline app/js/components/ConceptDisplay.jsx:21]: This code, and the contents of `UNSAFE_componentWillReceiveProps` below, literally did nothing. Only there to be confusing.

File: app/css/lab-results-view.scss [hunk 2]
```diff
@@ -151,7 +151,12 @@ i.scale:hover {
     100% { transform: rotate(360deg); }
 }
 
-
+@media print {
+  .ReactTable {
+    max-width: 90%;
+    margin-left: 5%;
+  }
+}
```
Comment (brandones) [inline app/css/lab-results-view.scss:159]: This adds the margin.

File: app/js/components/ConceptDisplay.jsx [hunk 3]
```diff
@@ -12,27 +12,19 @@ const {
...
 
   UNSAFE_componentWillReceiveProps(nextProps) {
-    const { concept } = this.state;
-    if (nextProps.concept !== concept) {
-      this.setState({
-        concept,
-      });
+    const { onLoaded } = this.props;
+    if (!R.isEmpty(nextProps.concept)) {
+      onLoaded();
```
Comment (brandones) [inline app/js/components/ConceptDisplay.jsx:27]: The `ConceptDisplay` reports its loading status back up to the `Cell` which reports its loading status back up to the table. The table resolves a promise when all the cells are loaded.

File: app/js/components/LabResultsList.jsx [hunk 4]
```diff
@@ -465,7 +501,7 @@ export class LabResultsList extends PureComponent {
                 id="app.labResultsList.title"
                 defaultMessage="Lab Test Results" />
             </h2>
-
+            {this.state.isPrinting ? <LocalizedPatientHeader /> : null}
```
Comment (brandones) [inline app/js/components/LabResultsList.jsx:504]: This adds the patient banner.

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @brandones !

**Pull request merged**